StereoKitC/xr_backends/extensions/palm_pose.cpp

```diff
@@ -54,10 +55,17 @@ void xr_ext_palm_pose_on_profile(void*, xr_interaction_profile_t* ref_profile) {
 	// This function will only get called if the system was successfully
 	// initialized.
 
-	// EXT_palm_pose applies to all interaction profiles that use /user/hand/*
-	// paths, so we need to check if the top level path matches.
-	if (string_eq("/user/hand/left",  ref_profile->top_level_path)) ref_profile->binding[ref_profile->binding_ct++] = { xra_type_pose, input_pose_l_palm, "palm_ext/pose" };
-	if (string_eq("/user/hand/right", ref_profile->top_level_path)) ref_profile->binding[ref_profile->binding_ct++] = { xra_type_pose, input_pose_r_palm, "palm_ext/pose" };
+	if (backend_openxr_ext_enabled(XR_KHR_MAINTENANCE1_EXTENSION_NAME)) {
```

Comment:
I wasn't sure which one should take precedence, so I defaulted to the "newer" one.